backend/app/assets/stylesheets/spree/backend/shared/_typography.scss

```diff
@@ -1,5 +1,10 @@
 // Base
 //--------------------------------------------------------------
+
+html {
+  font-size: $font-size-root;
```

Comment:
That changes the size of our `rem`, which I don't want to do. It's awkward that our base font size doesn't match a `rem`, making `rem` pretty useless if it doesn't match up with anything else on the page. If we're setting `$font-size-base` to a `px` value anyways we're not improving accessibility in any way.

I appreciate the argument that using the browser's default font size is a good thing. But if we aren't (and I think it's too big a change for this PR) the way bootstrap used to do it, and the way we're doing it here, by setting a size on the root element, is best.